doc/source/cluster/cluster_under_construction/running-applications-on-ray-clusters/job-submission/index.md

```diff
@@ -1,28 +1,41 @@
-(ref-deployment-guide-under-construction)=
+(ray-jobs-under-construction)=
 
-# Job Submission
+# Ray Jobs
 :::{warning}
 This page is under construction!
 :::
 
+Once you have deployed a Ray cluster (on [VMs](ref-cluster-quick-start-vms-under-construction) or [Kubernetes](kuberay-quickstart)), you are ready to run a Ray application!
+
+The recommended way to run a Ray application on a Ray Cluster is to use *Ray Jobs*.
+This mechanism allows you to submit locally developed applications to a remote Ray Cluster for execution.
+It simplifies the experience of packaging, deploying, and managing a Ray application.
+
 :::{note}
-(cade@) This was migrated from the old deployment guide
+This component is in **beta**. APIs may change before becoming stable.
```

Comment:
@architkulkarni just checking -- are jobs still beta in 2.0.0?

Reply:
Yes, confirmed with him that it is.